src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -162,7 +162,8 @@ export default class NameTranslation extends Mixins(CommonMixin) {
   @Prop({ default: () => { return [] as [] } })
   private nameTranslations!: NameTranslationDraftIF[]
 
-  private actionTypes = ActionTypes
```

Comment:
Was not used. But if it was needed (ie, in the template) then it should be:
`private ActionTypes = ActionTypes`

In other words, let's keep types capitalized.